backends-common/postgres/src/main/java/org/apache/james/backends/postgres/quota/PostgresQuotaModule.java

```diff
@@ -72,7 +72,7 @@ interface PostgresQuotaLimitTable {
                 .column(QUOTA_TYPE)
                 .column(QUOTA_LIMIT)
                 .constraint(DSL.constraint(PK_CONSTRAINT_NAME).primaryKey(QUOTA_SCOPE, IDENTIFIER, QUOTA_COMPONENT, QUOTA_TYPE))))
-            .supportsRowLevelSecurity();
+            .disableRowLevelSecurity();
```

Comment:
Why that change? Did I miss something?

Reply:
I guess because it in `postgres-common` module, 
maybe we should only RLS on mailbox module?

Reply:
I have asked in https://github.com/linagora/james-project/issues/4972 and @chibenwa said that only mailbox annotation, message, mailbox message need RLS

Reply:
He said RLS tests needed... TESTS. Does that mean we should disable them for quota and RRT?

It's confusing, and I prefer to be sure: @chibenwa ? :)

Reply:
:+1: It should be splitting smaller commit

Reply:
No personnal data on quota TBH and quota :

 - Are out of the mailbox so linking a domain is not meaningful
 - Quota limits are defined bu user, domain and globally. RLS do not make sens when applied globally.
 
 BTW @hungphan227 "Benoit said" is a justification - argument of authority - that shall not be used. Ideally we should discuss enough so that every body understand why a decision is taken. And be able to properly argument it.